content/influxdb/cloud/write-data/replication.md

```diff
@@ -2,7 +2,7 @@
 title: Replicate data from InfluxDB OSS to InfluxDB Cloud
 weight: 106
 description: >
-  Use InfluxDB replication streams to replicate all data written to an InfluxDB OSS
```

Comment:
do we want to keep the "replication streams" verbiage as well? 

eg: replication streams (aka InfluxDB Edge Data Replication)

Reply:
hey @samhld, to be sure, I'm combing thru, and see with this change above ^^ we no longer want to specify "instance to InfluxDB Cloud" (see line 6 below)

I'll remove it for now, but our heading indicates this particular doc is specific instructions for writing to Cloud